launch_testing/launch_testing/launch_test.py

```diff
@@ -54,10 +54,12 @@ def main():
                         default=False,
                         help='Show arguments that may be given to the test file.')
 
-    parser.add_argument('--disable-ros-isolation',
+    # TODO(hidmic): Provide this option for rostests only.
+    parser.add_argument('-i', '--isolated',
                         action='store_true',
                         default=False,
-                        help='Do not set a ROS_DOMAIN_ID.  Useful for debugging ROS tests')
+                        help=('Isolate tests using a custom ROS_DOMAIN_ID.'
+                              'Useful for test paralellization.'))
```

Comment:
Spelling: parallelization?

Reply:
Oh my. Fixed in https://github.com/ros2/launch/pull/240/commits/cbfa7dee7c7641a4363e3ea9dc26bfa27d0d8406.